testssl.sh

```diff
@@ -8445,8 +8451,8 @@ check_tls_serverhellodone() {
           [[ $remaining -lt 10 ]] && return 1
 
           tls_content_type="${tls_hello_ascii:i:2}"
-          [[ "$tls_content_type" != "15" ]] && [[ "$tls_content_type" != "16" ]] && \
-               [[ "$tls_content_type" != "17" ]] && return 2
+          [[ "$tls_content_type" != "14" ]] && [[ "$tls_content_type" != "15" ]] && \
+               [[ "$tls_content_type" != "16" ]] && [[ "$tls_content_type" != "17" ]] && return 2
```

Comment:
a case statement (which matches 14 to 17 and do nothing, for all other a ``return 2``) would be more readable